scripts/asm-installer/tests/run_basic_suite

```diff
@@ -85,6 +84,19 @@ main() {
   verify_demo_app "${GATEWAY}" || SUCCESS=1
   set -e
 
+  if [[ "${SUCCESS}" -eq 1 ]]; then
+    echo "Failed to verify, restarting and trying again..."
```

Comment:
Maybe comment some justification for why we try twice, versus considering this an error. (I assume flakiness, but this could cover a real issue.)